opennms-doc/guide-install/src/asciidoc/text/opennms/windows.adoc

```diff
@@ -0,0 +1,99 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-install-opennms-windows]]
+=== Install on Microsoft Windows Systems
+
+_OpenNMS_ is mostly developed on Unix/Linux based systems, nevertheless it is possible to install the platform on _Microsoft Windows_ operating systems.
+To install the application a graphical installer is provided and can be used to install in the _Microsoft Windows_ environment.
```

Comment:
"install in the _Microsoft Windows_" -> "install _OpenNMS_ in the _Microsoft Windows_"